Engine/Module/ModuleFileSystem.cpp

```diff
@@ -140,14 +140,17 @@ bool ModuleFileSystem::Exists(const std::string& path) const
 bool ModuleFileSystem::Remove(Path* path)
 {
 	assert(path != nullptr);
-
+	if (path->IsMeta())
+	{
+		std::string imported_file_path = path->GetFullPath().substr(0, path->GetFullPath().find_last_of("."));
+		PHYSFS_delete(imported_file_path.c_str());
+	}
```

Comment:
It feels strange that this specific case is on a generic remove. I suppose that this case is for `PanelProjectExplorer`, when it's removing a file showed in explorer (correct me if Im wrong). 

Maybe this code could be in other class , like `MetafileManager`, where all the deletion process of a metafile is done (deleting the metafile, the asset and also the bianries generated)

Reply:
Yes, I was going to do the deleted and I added this but since it is very problematic with things like this I will let you do it instead

Reply:
Ok!